demo/admin/src/warnings/WarningsGrid.tsx

```diff
@@ -57,12 +58,17 @@ function WarningsGridToolbar() {
     );
 }
 
-export function WarningsGrid() {
+interface WarningsGridProps {
+    warningMessages?: Record<string, MessageDescriptor>;
```

Comment:
Have you considered using ReactNode instead for MessageDescriptor? This would allow `<FormattedMessage ... />`, but also string (or any other component) for projects that don't use FormatJS

Reply:
changed here: a932bbd5338e90ee05e84733f3e4404eeba6e7e4